packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -28,19 +31,24 @@ const EDiscovery = WebexPlugin.extend({
   /**
    * Creates a compliance report with a specific set of search parameters
    * @param {Object} reportRequest - A set of criteria for determining the focus of the search
+   * @param {Object} options - can specify this option :
+   *                           timeoutMs - connection timeout in milliseconds, defaults to 10s
```

Comment:
To get this option to display in jsdoc readers, you can change this line to:
`* @param {number} options.timeoutMs - connection timeout in milliseconds, defaults to 10s`

Reply:
updated the jsdoc now, thanks for the tip